app/src/full/java/org/openobservatory/ooniprobe/common/FirebaseMessagingService.java

```diff
@@ -17,7 +21,15 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
 		// decode message data and extract meaningful information from it: title, body, badge, etc.
 		CountlyPush.Message message = CountlyPush.decodeMessage(remoteMessage.getData());
 
-		Boolean result = CountlyPush.displayMessage(getApplicationContext(), message, R.drawable.notification_icon, null);
+		Intent notificationIntent = null;
+		if (!message.has("c.l")) {
+			notificationIntent = new Intent(this, MainActivity.class);
+			notificationIntent.putExtra(MainActivity.NOTIFICATION_DIALOG, "yes");
+			notificationIntent.putExtra("title", "title");
```

Comment:
Is it intended here that the `"title"` extra attribute has the `"title"` string as value?